### ISSUE TITLE
ДЗ2

### ISSUE THREAD

**Issue by anrom7** [JanusDG/Homework_0_2nd_term#2]
Про що йдеться - Точність моделі можна обрати ?
Вимоги ще раз перегляньте.
Спільної роботи не бачу.

**Comment by JanusDG** [comment]
Слава Ісусу Христу!
Про точність моделі - йде мова про те що можна заново і заново навчати
модель, поки точність не досягне певного значення.
Командна робота заключається в тому що ми об'єднали тематики в одне
дослідження що буде містити у собі аналіз та статистику даних про післні що
доступні у Deezer. Ми одночасно не працюємо над чимось одним, а намагаємось
розглянути дані з різних сторін. Моя частина роботи заключається в тому щоб
дослідити дані за допомогою модені, а мого партнера - дослідити кожнен з
параметрів окремо. Все це планується об'єднати в один висновок, та надати
більш різноманітну статичтику для користувача. Частина з цього була описана
в результаті виконання дз 2.
Дякую за відгук, сподіваюсь з нашою роботою все гаразд. Якщо це не так
будемо вдячні за пораду про те що можна щось покращити.

З повагою,
Білусяк Дмитро

**Comment by anrom7** [comment]
Можна безкінечно тренувати модель і не досягти вказаної точності.
Не впевнений що користувач повинен вводити точність.

**Comment by JanusDG** [comment]
Так, згоден, справді не потрібно давати користувачу таку можливість. Ми
переглянули ваше зауваження і не добавляли даний функціонал в кінцеву
реалізацію. Дякую!
